libs/dao-data/src/subgraph/queries-lens/profiles.generated.ts

```diff
@@ -0,0 +1,345 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import * as Types from '../schema-lens.generated';
+
+import { TypedDocumentNode as DocumentNode } from '@graphql-typed-document-node/core';
+export type ListProfileQueryVariables = Types.Exact<{
+  memberAddress: Types.Scalars['EthereumAddress'];
+}>;
+
+export type ListProfileQuery = {
+  profiles: {
+    items: Array<{
+      id: any;
+      name?: string | undefined;
+      bio?: string | undefined;
+      followNftAddress?: any | undefined;
+      metadata?: any | undefined;
+      isDefault: boolean;
+      handle: any;
+      ownedBy: any;
+      picture?:
+        | {
```

Comment:
never seen this syntax before, is the initial `|` needed here?

Reply:
it's generated so i can't really control that.